ticker/src/main/java/com/robinhood/ticker/TickerView.java

```diff
@@ -149,6 +150,23 @@ protected void init(Context context, AttributeSet attrs, int defStyleAttr, int d
             textPaint.setShadowLayer(styledAttributes.shadowRadius, styledAttributes.shadowDx,
                     styledAttributes.shadowDy, styledAttributes.shadowColor);
         }
+        if (styledAttributes.textStyle != 0) {
+            final int textStyle = styledAttributes.textStyle;
+            int flags = textPaint.getFlags();
+            if ((textStyle & AssistStructure.ViewNode.TEXT_STYLE_BOLD) != 0) {
+                flags |= Paint.FAKE_BOLD_TEXT_FLAG;
+            }
+            if ((textStyle & AssistStructure.ViewNode.TEXT_STYLE_ITALIC) != 0) {
+                textPaint.setTextSkewX(-0.25f);
```

Comment:
good point, TextView sets a flag on the typeface object so changed approach to that way.